requirements/edx/github.in

```diff
@@ -49,25 +63,15 @@
 #     re-install the package each time, and can be useful when working with two
 #     repos before picking a version number. Don't use 0.0 on master, only for
 #     tight-loop work in progress.
-
-
-# Python libraries to install directly from github
-
-# Third-party:
--e git+https://github.com/openedx/django-wiki.git@1.1.1#egg=django-wiki
--e git+https://github.com/openedx/olxcleaner.git@2f0d6c7f126cbd69c9724b7b57a0b2565330a297#egg=olxcleaner
+#
+#   * Alphabetize dependencies by DIST-NAME.
+git+https://github.com/openedx/blockstore.git@1.2.5#egg=blockstore==1.2.5
+git+https://github.com/openedx/codejail.git@3.3.0#egg=edx-codejail==3.3.0
+git+https://github.com/openedx/codejail.git@3.1.3#egg=codejail==3.1.3
```

Comment:
I messed up when rebasing this PR and ended up merging this with two different installations of the same package 😬 

Fix is here: https://github.com/openedx/edx-platform/pull/31188